app/src/main/res/layout/activity_list_shops.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:tools="http://schemas.android.com/tools"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        tools:context=".activity.ListShopsActivity">
+    <androidx.recyclerview.widget.RecyclerView
+            android:layout_width="395dp"
+            android:layout_height="715dp" app:layout_constraintEnd_toEndOf="parent"
```

Comment:
The reason this will be going off the screen is because the width and height are probably too large for the device you are testing on. You could do two things:

1. Set the width and height to match_parent this will make it the same height as the parent.
2. The second thing is that with the constraints you have on the recyclerview you could set the width and height to 0dp. This would work as that is shorthand for match_constraint which means the view will expand to fill the available space for its defined constraints